documentation/operators/src/legal/isp-list.md

```diff
@@ -0,0 +1,25 @@
+# Where to host your `nym-node`?
+
+```admonish info
```

Comment:
why do we need this if we already have this license for the rest of the docs?

Reply:
true, there was something with the community legal pages etc to be more xplicit, but I am happy to take it down. your call.